queue backlog

cmccully opened the pull request:
Added initial frame rejection to the listener to hopefully fix the backlog of messages on the archived_fits queue.

cmccully committed:
Added initial frame rejection to the listener to hopefully fix the backlog of messages on the archived_fits queue.

cmccully committed:
added a nasty hack to get around the fact that all these messages have db1 in them.

eheinrich commented on nrespipe/tasks.py:
```diff
@@ -70,32 +68,12 @@ def run_idl(idl_procedure, args, data_reduction_root, site, nres_instrument):
-
-    if filename_is_blacklisted(filename):
-        logger.debug('Filename does not pass black list. Skipping...', extra={'tags': {'filename': filename}})
+def process_nres_file(file_info, data_reduction_root_path, old_db_address):
```
Is the `old_db_address` being used somewhere?

eheinrich commented on nrespipe/utils.py:
```diff
@@ -651,3 +651,44 @@ def ingest_file(file_path):
+        should_process = False
+
+    if not should_process:
+        dbs.set_file_as_processed(filename, checksum, file_info.get('frameid'), db_address)
```
Just checking, this isn't the datastore that you said could only handle one update at a time right?

cmccully committed:
Fix to l1pubdat logic to work on leap years. Path fixes in the jenkins file.

cmccully committed:
Made logging in the ingester more verbose.

cmccully committed:
Another path fix.

cmccully committed:
Typo fix.

cmccully committed:
Fix to the db logic in the listener when a checksum changes.

cmccully committed:
Fix to the db logic in the listener when a checksum changes.

cmccully committed:
Fix to the db logic in the listener when a checksum changes.

Commit:
Update lco-ingester dependency version

The lco-ingester library has been updated to version 2.1.14, which
itself contains an update to one of its dependencies,
opentsdb-python-metrics, which publishes metrics data on the standard
HTTP port 80 by default, rather than using a custom port number.

This behavior is also now configurable, rather than being hard-coded
into the library.

With this change, the LCO IT staff will be able to migrate the OpenTSDB
server to our newer hardware platform.

mgdaily committed:
Merge branch 'fix/queue_backup' into upgrade-ingester-library

mgdaily committed:
Pin version of astroquery.

When merging changes from the base branch, I accidentally removed the pinning on astroquery. This commit re-instates that.

mgdaily committed:
Merge pull request #41 from LCOGT/upgrade-ingester-library

Update lco-ingester dependency version

cmccully committed:
Fixes to refine traces.

cmccully committed:
Merge branch 'fix/queue_backup' of github.com:lcogt/nres-pipe into fix/queue_backup
